tests/test_api_path_traversal.py

```
@@ -0,0 +1,157 @@
+
+    clear_spec_manager()
+    set_spec_manager(MockSpecManager(runs_root))
+
+    yield
+
+    clear_spec_manager()
+
+def run_async(coro):
+    return asyncio.run(coro)
+
+def test_events_stream_traversal(mock_app_context):
```

Ruff (0.15.1)

[warning] Unused function argument: `mock_app_context`

(ARG001)